source/documentation/monitoring_apps/pentest.md

```diff
@@ -0,0 +1,44 @@
+# Monitoring apps
+
+## Penetration testing
+
+You must penetration test your app to make sure it’s secure. Because GOV.UK PaaS runs on Amazon Web Services (AWS), you need permission from AWS to penetration test your app. 
+
+To request permission, the PaaS team must complete and submit the AWS Vulnerability / Penetration Testing Request Form. The PaaS team completes this form on your behalf because this form requires root credentials for the AWS portal.
+
+You must send the information specified in the tables below to the PaaS team at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) so that they can complete the form.
+
+|Supplied by penetration tester|
+|:---|
+|Tester phone number|
+|Emergency email and phone number contact details|
+|Confirmation that the requests will originate from the tester’s office|
+|Confirmation that the tester has signed a non-disclosure agreement (NDA) with AWS
+|The origin IPs|
+|Peak bandwidth that the tests will consume in gigabits per second|
+|Peak number of requests per second the tests will perform|
+|Start and end dates and times of the test in the format YYYY-MM-DDTHH:MM:SSZ e.g. 2017-09-26T09:00:00Z|
+|The metrics you should measure to decide if the test succeeds or fails|
+|Confirmation that it is possible to stop the test immediately if there is an issue|
+
+|Supplied by tenant|
+|:---|
+|Specify if the tester is a third-party company|
+|Details on the purpose of the test and what is being covered|
+
+You should supply as much information as possible to minimise delays in this process.
+
+AWS responds to permission request forms within two business days. The PaaS team will forward the AWS response to you within two further business days of receiving it.
+
+If your request is approved, you will receive an authorisation number. Your penetration test is authorised for the dates and times specified in the form.
+
+If your request is not approved, the PaaS team will liaise with you and AWS to solve any outstanding issues. 
+
+Note that you can only penetration test your app; you cannot penetration test the platform. 
```

Comment:
I feel like this line should be at the *start* of this section not the end as it seems like quite an important distinction.

For example the intro could be something like:

"We perform our own penetration testing of the GOV.UK PaaS platform itself, but you must perform penetration testing of your own apps to ensure they too are secure..."